common/scripted_effects/wc_mana_effects.txt

```diff
@@ -0,0 +1,62 @@
+﻿wc_regenerate_mana_effect = {
+	if = {
+		limit = { has_variable = wc_current_mana_variable }
+
+		change_variable = { name = wc_current_mana_variable add = wc_mana_regeneration_value }
+		wc_recalculate_mana_effect = yes
+	}
+	else = { set_variable = { name = wc_current_mana_variable value = wc_max_mana_value } }
+
+	wc_recalculate_prowess_from_mana_effect = yes
```

Comment:
```suggestion
	wc_restore_AMOUNT_of_mana = { AMOUNT = wc_mana_regeneration_value }
```

Reply:
Should be fixed during https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/162#discussion_r664780542.